src/plugins/data/common/index_patterns/constants.ts

```diff
@@ -15,3 +15,15 @@ export const RUNTIME_FIELD_TYPES = [
   'boolean',
   'geo_point',
 ] as const;
+
+/**
+ * Used to determine if the instance has any user created data by filtering data created by Fleet server
+ */
+export const KNOWN_FLEET_ASSETS = {
+  LOGS_INDEX_PATTERN: 'logs-*',
+  METRICS_INDEX_PATTERN: 'metrics-*',
+  INDEX_PREFIXES_TO_IGNORE: [
```

Comment:
I'm confused why these require special handing when they start with a `.` and therefore should be treated as system indices

Reply:
This appears as data stream as `metrics-elastic_agent` so it isn't ignored by client code where we filter system indices. 
So I changed that clientside filter: 

https://github.com/elastic/kibana/blob/3334fd5278b68627cb14b4eb326956e2d3ee5747/src/plugins/index_pattern_editor/public/components/empty_prompts/empty_prompts.tsx#L37-L40

I am happy to remove the one if we decide it not worth it: https://github.com/elastic/kibana/pull/108887#discussion_r691286896

Reply:
Data stream indices (eg. `.ds-logs`) will be queried when searching on a pattern (eg. `logs-*`) that matches a data stream.